### PR TITLE
Add MINSIGSTKSZ and SIGSTKSZ definitions for MacOS

### PR DESCRIPTION
**Pull request by jedisct1**

### CODE REVIEW

**Comment by alexcrichton** [comment]
Thanks for the PR here, but [right now there's not many maintainers to help merge this](https://github.com/rust-lang/libc/issues/1657), so it may take awhile.

**Comment by JohnTitor** [comment]
Looks good! @jedisct1 could you rebase and resolve conflicts?
r? @JohnTitor

**Comment by jedisct1** [comment]
Rebased!

File: src/unix/bsd/apple/mod.rs [hunk 1]
```diff
@@ -3091,6 +3091,11 @@ pub const SF_IMMUTABLE: ::c_uint = 0x00020000;
 pub const SF_APPEND: ::c_uint = 0x00040000;
 pub const UF_HIDDEN: ::c_uint = 0x00008000;
 
+// sys/signal.h
+
+pub const MINSIGSTKSZ: ::size_t = 32768;
+pub const SIGSTKSZ: ::size_t = 131072;
```
Comment (JohnTitor) [inline src/unix/bsd/apple/mod.rs]: Oh, `SIGSTKSZ` is already defined at L2501:

https://github.com/rust-lang/libc/blob/d692ac288e734ff51094eec4e3f0c9995727f5c2/src/unix/bsd/apple/mod.rs#L2501

Could you just move `MINSIGSTKSZ` and remove `SIGSTKSZ` from the change?

**Pull request merged**